models/nc-news-models.js

```diff
@@ -56,4 +56,15 @@ exports.patchVotes = ({newVote}, article_id) => {
           } 
         return rows[0]
         })
+}
+
+exports.deleteCommentById = (comment_id) => {
+    return db
+    .query(`SELECT * FROM comments WHERE comment_id = $1`, [comment_id])
+    .then(({rows}) => {
+        if(!rows.length) {
+            return Promise.reject({status: 404, msg: "Comment does not exist"})
+        }
+    return db.query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id])
```

Comment:
You can achieve this same functionality by RETURNING the deleted comment and if there is non then you know it wasn't found

Reply:
Ooo nice, I didn't know that cheers!!